When randomizing user film dictionary --> was filtering film_id's that were 1 higher than
user dict values (aka user films)

(venv) (base) mosesfarley@Mosess-MacBook-Air metacritic_top10 % /Users/mosesfarley/metacritic_top10/venv/bin/python /Users/mosesfarley/metacritic_top10
/metacritic_films/cosine_output_TEST.py
{417, 1605, 968, 1611, 912, 1759, 1587, 1111, 446, 671}  --> COMPARE THIS 



       film_id  critic_id  ranking ---> TO FILM_ID COLUMN VALUES HERE
21         672        168        2
2939      1588        177       10
3922       447        624        3
7364      1606        185        5
7480      1606          8        1
...        ...        ...      ...
12449      969        176       10
12615     1612        121        6
12756     1612         91        7
12879     1612        601       10
16299     1760         20       10


VS. with pre-assigned film list (Peter Travers) 

[67, 86, 88, 112, 168, 196, 222, 301, 325, 333] --> THIS film list


       film_id  critic_id  ranking --> Matches the film_id's here. 
47         325        185        8
56          67        228        7
63         325        140        4
76         196        333        7
89          67        266       10
...        ...        ...      ...
18011      222        625        2
18052      222        447        3
18063      222        539        4
18204      222        236        5
18268      222        591        9

[712 rows x 3 columns]

WHY?
1. is it dictionary vs list? 


**SOLUTION: 
titles.at[] was looking at index value rather than film_id!